.travis.yml

```diff
@@ -55,3 +55,6 @@ matrix:
     - script: make licenses BINDIR="$(dirname $(which python))"
       env:
         - SUITE=Licenses
+    - script: make translations BINDIR="$(dirname $(which python))"
```

Comment:
The alternative could look like this: https://github.com/pypa/warehouse/issues/7409#issuecomment-591443217. Having an up-to-date translations source is more useful in master after the PR merge, not in PRs themselves.